yellowbrick/text/postag.py

```diff
@@ -139,16 +144,34 @@ def fit(self, X, y=None, **kwargs):
         self : instance
             Returns the instance of the transformer/visualizer
         """
-        # TODO: add support for other tagsets?
-        if self.tagset == "penn_treebank":
-            self.pos_tag_counts_ = self._penn_tag_map()
-            self._handle_treebank(X)
-
-        elif self.tagset == "universal":
-            self.pos_tag_counts_ = self._uni_tag_map()
-            self._handle_universal(X)
+        if(self.stack == True and y == None):
+            raise YellowbrickValueError("Specify y for stack=True")
+        
+        labeled = []
+        if(y!=None):
+            for i in range(len(y)):
+                labeled.append((X[0][i],y[i]))
+        self.label_ = set(map(lambda x:x[1], labeled))
```

Comment:
I think it's important that we keep track of the unique labels in `y` if they're passed in - but I would like to make this learned attribute consistent with the other classification visualizers. Would you mind calling this `self.labels_`?